### PR TITLE
os/bluestore/bluefs: add inspection of bluefs files

### PR DESCRIPTION
**Pull request by aclamk**
This allows to see which bluefs files are located on which device. 
It was useful to analyze https://bugzilla.redhat.com/show_bug.cgi?id=1801203 .

### CODE REVIEW

File: src/os/bluestore/BlueFS.cc [hunk 1]
```diff
@@ -112,6 +115,33 @@ class BlueFS::SocketHook : public AdminSocketHook {
     } else if (command == "bluestore bluefs stats") {
```
Comment (howard0su) [inline src/os/bluestore/BlueFS.cc]: shall we remove bluestore here to keep consistent?

File: src/os/bluestore/BlueFS.cc [hunk 2]
```diff
@@ -112,6 +115,33 @@ class BlueFS::SocketHook : public AdminSocketHook {
...
+          sizes.resize(bluefs->bdev.size());
+          for(auto& i : r.second->fnode.extents) {
+            sizes[i.bdev] += i.length;
+          }
+          for (size_t i = 0; i < sizes.size(); i++) {
+            if (sizes[i]>0) {
+	      if (i < sizeof(devnames) / sizeof(*devnames))
+		f->dump_int(devnames[i], sizes[i]);
+	      else
+		f->dump_int(("dev-"+to_string(i)).c_str(), sizes[i]);
```
Comment (ifed01) [inline src/os/bluestore/BlueFS.cc]: you can prefill devnames with these 'dummy' names and simplify the code here...

**Comment by tchaikov** [comment]
@aclamk needs rebase

**Comment by tchaikov** [comment]
@aclamk ping?

**Comment by tchaikov** [comment]
http://pulpito.ceph.com/kchai-2020-05-21_10:34:02-rados-wip-kefu-testing-2020-05-21-1652-distro-basic-smithi/

the failures are either tracked or fixed by

- https://tracker.ceph.com/issues/44595
- https://tracker.ceph.com/issues/45647
- https://tracker.ceph.com/issues/45631
- https://tracker.ceph.com/issues/44990
- #35139

**Pull request merged**